CHANGES.txt

```diff
@@ -4,7 +4,8 @@ Changes
 2.2 (unreleased)
```

Comment:
This should be version 3.0. This is a **major** breaking change and should break approximately everyone using this package.

Reply:
That's good, though probably "switched timezone provider" is maybe a little under-stated here. I imagine that people won't realize that this completely changes the API of this package. There should probably be big bold text like, "If this change doesn't break you there's a good chance you were using this package wrong in the first place", and possibly a migration guide like the one in [`pytz-deprecation-shim`](https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html#migration-guide).

You could even use that one, or recommend that people who want a slower deprecation use `pytz-deprecation-shim`, either [`wrap_zone`](https://pytz-deprecation-shim.readthedocs.io/en/latest/api.html#pytz_deprecation_shim.wrap_zone) or [`timezone`](https://pytz-deprecation-shim.readthedocs.io/en/latest/api.html#pytz_deprecation_shim.timezone).